triggers/incident-triggered/handler.py

```diff
@@ -0,0 +1,27 @@
+from relay_sdk import Interface, WebhookServer
+from quart import Quart, request, render_template_string
+
+import logging, sys
+
+relay = Interface()
+app = Quart('incident-triggered')
+
+logging.getLogger().setLevel(logging.INFO)
+
+@app.route('/', methods=['POST'])
+async def handler():
+    data = await request.form
+
+    event = { 'incident': 0 }
+
+    # handle incident_id a bit special since it's the main key, the rest of the
+    # payload may change so just stuff it in a top-level key unmodified
+    for field, value in data.items():
+        event[field] = value.strip('"')
```

Comment:
This is so strange. I wonder what encoding logic they're using on their side here.